com.heroicgameslauncher.hgl.yml

```diff
@@ -107,7 +107,7 @@ modules:
     sources:
       - type: git
         url: https://github.com/KhronosGroup/Vulkan-Tools.git
-        tag: sdk-1.3.224.1
+        tag: sdk-1.3.261.1
```

Comment:
Required for the flaptak to compile (the flatpak's headers have been updated and stuff has been renamed, etc etc it's not compatible without an update)